src/components/Store/StoreEdit.vue

```diff
@@ -108,7 +108,7 @@
             </q-btn>
             <q-btn
               type="button"
-              @click="$emit('cancel')"
+              @click="$router.push('/group/' + $route.params.groupId)"
```

Comment:
This works, but I would say that it does the wrong thing and it does it at the wrong place. 

- I think it should go back to the previous page, not the the group home page.
- I wouldn't trigger the router push here, but in the parent component. That keeps the `StoreEdit` component more isolated.
- We always use named routes instead of string concat

As the parent component is a connect component, you'll need something similar to this: https://github.com/yunity/karrot-frontend/blob/master/src/pages/GroupGallery.vue#L14